client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-assembler-preview.tsx

```diff
@@ -16,6 +16,16 @@ import type { Pattern } from './types';
 import type { Design } from '@automattic/design-picker';
 import './pattern-assembler-preview.scss';
 
+const inlineCss = `
+	.site-footer-container {
+		margin-block-start: auto;
+	}
+
+	.wp-site-blocks {
+		min-height: 100vh;
```

Comment:
The viewport height is set [here](https://github.com/Automattic/wp-calypso/blob/trunk/packages/design-picker/src/utils/designs.ts#L32), so I think we just need to set the value to `undefined` in [this file](https://github.com/Automattic/wp-calypso/blob/trunk/client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-assembler-preview.tsx#L79)

Reply:
Thanks 🤔 It's not working with undefined or null so I'll leave it.

Reply:
The reason might be it will fallback to default 😂

```
viewport_height: options.viewport_height || DEFAULT_VIEWPORT_HEIGHT,
```